File: CleanCodeJava/app/src/main/java/io/chuumong/cleancodejava/data/api/CityService.java
package io.chuumong.cleancodejava.data.api;

import io.chuumong.cleancodejava.data.model.CityList;
import retrofit2.http.GET;
import rx.Observable;

/**
 * Created by JongHunLee on 2017-02-22.
 */

public interface CityService {

    @GET("v1/city")
    Observable<CityList> getCitys();

}

File: CleanCodeKotlin/app/src/main/java/io/chuumong/cleancodekotlin/data/api/CityService.kt
package io.chuumong.cleancodekotlin.data.api

import io.chuumong.cleancodekotlin.data.model.CityList
import retrofit2.http.GET
import rx.Observable

/**
 * Created by JongHunLee on 2017-02-23.
 */
interface CityService {
    @GET("v1/city")
    fun getCitys(): Observable<CityList>
}